build.sbt

```diff
@@ -13,9 +13,7 @@ lazy val root = (project in file("."))
     majorVersion := 0,
```

Comment:
extract this to global level like this `ThisBuild / majorVersion := 0` per sbt plugin